CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ endif()
 install(FILES testnet.template DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/eosio/launcher COMPONENT dev EXCLUDE_FROM_ALL)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/testnet.template ${CMAKE_CURRENT_BINARY_DIR}/etc/eosio/launcher/testnet.template COPYONLY)
 
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/eosio.version.in ${CMAKE_CURRENT_BINARY_DIR}/eosio.version.hpp)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/leap.version.in ${CMAKE_CURRENT_BINARY_DIR}/eosio.version.hpp)
```

Comment:
I don't like the inconsistency between the source filename and the generated file name.

However, since the generated file is included in to the installation package, it would not be appropriate to change this for 3.1 or 3.2 (only for main).

So it would be best to keep this eosio.version.in in this PR.

If we want to do some renaming in the `main` branch (in preparation for 4.0), then that is fine, but then the generated files should also be consistently renamed.

Reply:
Makes sense. I will move the applicable commits to a separate PR for `main` and change the generated names in the binary directories as well. I should also rename `eosio.version.hpp`.

Reply:
Reverted in commit https://github.com/AntelopeIO/leap/pull/358/commits/cb3ccfc739c14996b4301b0774ba78548fc7f4a2.